### ISSUE TITLE
bug for CudaLongTensor conversion

### ISSUE THREAD

**Issue by congling** [arunpatala/reddit.captcha#13]
Hi, 
   I've tried your sample. But I found that the code has some issues for tensor conversion 
   When it runs to these two lines, th will report 'invalid arguments CudaLongTensor CudaTensor' 
    lloss = lloss + YYb:eq(Yb):sum()
    lloss = lloss + YYb:eq(Yb):sum(2):eq(5):sum()
  I changed it into the following to make it work
    lloss = lloss + YYb:long():eq(Yb:long()):sum()
    lloss = lloss + YYb:long():eq(Yb:long()):sum(2):eq(5):sum()

  Btw, the training for 10K samples are very slow in my GTX 1080.  Do I need to set something else?
  Thanks

train .. 0.011111111111111
loss..250.91162276268
valid .. 3.38
train .. 6.5777777777778
valid .. 0
train .. 0.044444444444444
loss..248.19494605064
valid .. 3.98
train .. 8.1777777777778
valid .. 0
train .. 0.21111111111111

**Comment by congling** [comment]
BTW, I was using 'java -cp simplecaptcha-1.2.1.jar:. Main' to generate 10K samples in data/simpleRNN/ folder
